### PR TITLE
[TS] Add class to LinkProps type

### PR DESCRIPTION
**Pull request by niicojs**
Having a class is a common scenario which gives me error all the time.
Simple fix for that.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { VNode } from "hyperapp";
 interface LinkProps {
   to: string;
   location?: Location;
+  class?: string;
```
Comment (frenzzy) [inline index.d.ts]: maybe `[attributeName]: any` ?

Comment (niicojs) [reply index.d.ts]: I don't need it so didn't implement it. I don't know a way to be exhaustive here.

**Comment by slacktracer** [comment]
Sooo... Can we have it? =)

**Comment by jorgebucaran** [comment]
I'm having a technical issue and couldn't publish the package, but I've already contacted npm and asked them for help. A new bump should be ready to ship in a few hours, give or take. 👋

**Pull request merged**